quickjs.c

```diff
@@ -211,8 +211,6 @@ typedef enum {
     JS_GC_PHASE_REMOVE_CYCLES,
 } JSGCPhaseEnum;
 
-typedef enum OPCodeEnum OPCodeEnum;
```

Comment:
This also looks like a change that shouldn't be necessary. What exactly was the warning or error you got?